Backend/TimeTracker.Test/ProjectTests.cs

```diff
@@ -143,6 +147,74 @@ await projectController.UpdateProjectDetails(new ProjectDetailsDTO()
 
             Assert.IsTrue(userProjects.Data.Count == 1);
             Assert.IsTrue(userProjects.Data[0].Id == project.Data.Id);
+        }
+
+        [TestMethod]
+        public async Task RemoveUserFromProject()
+        {
+            var createResult = await projectController.CreateProject(new ProjectCreateDTO()
+            {
+                ClientName = "Test Client",
+                Description = "Test Description",
+                ProjectName = "Test Name",
+                Tags = new List<string>()
+                {
+                    "Test Tag 1",
+                    "Test Tag 2"
+                }
+            });
+
+            Assert.IsTrue(createResult.Success);
+
+            var project = await projectController.GetProjectById(createResult.Data);
+
+            Assert.IsTrue(project.Data.Description == "Test Description");
+
+            await projectController.UpdateProjectDetails(new ProjectDetailsDTO()
+            {
+                Description = "Updated Description",
+                ProjectId = createResult.Data
+            });
+
+            var projectUpdated = await projectController.GetProjectById(createResult.Data);
+
+            Assert.IsTrue(projectUpdated.Data.Description == "Updated Description");
+
+            await projectController.AddUserToProject(new AddUserToProjectDTO()
+            {
+                InviteCode = project.Data.InviteCode
+            });

```

Comment:
Take a look at the `TestAuthHelper.LogInUser` function. It has an optional parameter for the invite code, this will log you in as that new user and join them to the project. You will need to switch back to the teacher account after this call. The code to "Mock" login a user is 

```c#
var user = new ClaimsPrincipal(new List<ClaimsIdentity>()
{
    new ClaimsIdentity(new List<Claim>()
    {
        new Claim(ClaimTypes.NameIdentifier, THE USER ID OF THE PERSON YOU WANT TO LOGIN AS)
    })
});

foreach(var controller in controllers)
{
    controller.ControllerContext.HttpContext = new DefaultHttpContext()
    {
        User = user
    };
}
```

I pulled this from the `TestAuthHelpers.LogInUser` function.